make/Main.gmk

```diff
@@ -1289,7 +1300,7 @@ all-docs-bundles: docs-jdk-bundles docs-javase-bundles docs-reference-bundles
 # This target builds the test image
 test-image: prepare-test-image test-image-jdk-jtreg-native \
     test-image-demos-jdk test-image-libtest-jtreg-native \
-    test-image-lib-native
+    test-image-lib-native test-image-setup-aot
```

Comment:
Please restore the dependencies from upstream (reverting https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/714); building `test-image-lib` now succeeds and can return here.
```
test-image: prepare-test-image test-image-jdk-jtreg-native \
    test-image-demos-jdk test-image-libtest-jtreg-native \
    test-image-lib test-image-lib-native \
    test-image-setup-aot
```

Reply:
Restored `test-image-lib`.